modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewPage.kt

```diff
@@ -228,6 +234,8 @@ private fun WhatsNewSlumberStudiosPreview(
             state = UiState.Loaded(
                 feature = WhatsNewFeature.SlumberStudiosPromo(
                     promoCode = "PROMO",
+                    hasFreeTrial = false,
```

Comment:
Should we name this to `hasOffer` since we are going to have the intro offer too?

Reply:
Good point. I renamed the variable in cf971146a4a0d801cfcc40bbe0e870f610f10373. I removed `Start free trial` button text and displayed "Subscribe to Plus" for the free user. Any offer applicable will be visible on the upsell screen.

In 5cf5f6db754df745a42bdec4ef1c58b90720465c, I replaced the message for the free user based on updated text in Figma (G3TCvPyQ1gEXdxBKiiJ4aDeQ-fi-4347_3385):

<img width=320 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/6e67ca71-2158-4662-967a-d5b37e907e8d"/>